shipyard_blueprints/trello/shipyard_trello/cli/edit_card.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+import sys
+from trello import TrelloClient
+
+
+def get_args():
+    parser = argparse.ArgumentParser(description='Update a Trello card.')
+    parser.add_argument('--access_token', required=True, help='Your Trello access token.')
+    parser.add_argument('--api_key', required=True, help='Your Trello API key.')
+    parser.add_argument('--card_id', required=True, help='The ID of the Trello card you wish to update.')
+    parser.add_argument('--board_id', help='The ID of the Trello board you wish to move the card to.')
+    parser.add_argument('--list_name', help='The name of the Trello list you wish to move the card to.')
+    parser.add_argument('--card_name', help='The updated card name.')
+    parser.add_argument('--description', help='The description of the Trello card.')
+    parser.add_argument('--due_date', help='The due date of the Trello card.')
+    return parser.parse_args()
+
+
+def main():
+    args = get_args()
+    args_dict = vars(args)
+    trello = TrelloClient(access_token=args.access_token,
+                          api_key=args.api_key)
+    args_dict.pop('access_token')
+    args_dict.pop('api_key')
+
+    # Filter out blank values from update_ticket_args to avoid sending them to the Shortcut API
+    # and inadvertently overwriting valid ticket data.
+    update_card_args = {key: value for key, value in args_dict.items() if value not in (None, '')}
+
+    try:
+        trello.update_ticket(**update_card_args)
+
+    except Exception as error:
```

Comment:
It would be nice to identify what when wrong here, so we could have a custom exit code as well as let the user know

Reply:
It should log the error for the user without the large traceback message- is this ok for now ? My current thought process is this should be changed soon to be handled on the abstract class level